packages/obonode/obojobo-chunks-list/list-styles.test.js

```diff
@@ -298,4 +290,22 @@ describe('List Styles', () => {
 		expect(ls).not.toBe(other)
 		expect(ls.toDescriptor()).toEqual(other.toDescriptor())
 	})
+
+	test('all unordered lists dont have a start attribute', () => {
+		const ls = new ListStyles()
+		ls.init()
+		if (ls.type === 'unordered') {
```

Comment:
Given the expectation for `ls.type` to be one or the other, these conditions are unnecessary. In a worst-case scenario, not that I know how this might happen, this conditional could allow the test to pass even if the expectations are not met, since the expectations might not be checked at all.

Also this is just nitpicking, but if `unordered` is the default behavior, I would keep a check where it is not set (to make sure it's the default behavior) and also a separate check explicitly setting 'unordered' to make sure everything still reacts properly.

Reply:
I made these changes and took a closer look at the unit test for ordered which was filing with the removal of the condition. I learned `ls.init()` sets the style to unordered so I removed this line. **I added** `ls.get(indent) `as it calls other functions that seem to set up the list including adding the start attribute if it is ordered. I think this function would be called to create a list with defaults so I think it is a better way to test that these three unit tests function properly. 
A start attribute is not automatically added when `const ls = new ListStyles('ordered')` is created so this test was failing before I called `ls.get(indent) `which returns `getStyleWithDefaults(indent, this.type, this.styles[indent]) `as seen in `list-styles.js`